cmd/minishift/cmd/image/config_remove.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
+
+import (
+	"github.com/spf13/cobra"
+
+	"fmt"
+	"github.com/minishift/minishift/cmd/minishift/cmd/config"
+	"github.com/minishift/minishift/pkg/util/os/atexit"
+	"github.com/minishift/minishift/pkg/util/strings"
+)
+
+var (
+	removeConfiguredImageCmd = &cobra.Command{
+		Use:   "remove [image ...]",
+		Short: "Removes the specified images from the list of configured images for import and export.",
+		Long:  "Removes the specified images from the list of configured images for import and export.",
+		Run:   removeConfiguredImage,
+	}
+)
+
+func removeConfiguredImage(cmd *cobra.Command, args []string) {
```

Comment:
Some of the logic here is duplicated with `addConfiguredImage`. I would suggest to combine reading and writing logic as utility, and deal with 'append' or 'remove' in a different way.

Note: This is a separate issue to the one with the placement of these commands.

Reply:
> Some of the logic here is duplicated with addConfiguredImage. 

I can extract a helper.

> I would suggest to combine reading and writing logic as utility, and deal with 'append' or 'remove' in a different way.

Not sure I am following.